gradle/code-coverage.gradle

```diff
@@ -79,10 +79,11 @@ tasks.withType(JacocoReport).configureEach {
 
   reports {
     xml.enabled true
-    // Code coverage report in HTML format is on demand， since it takes up 100+MB of disk space.
-    if (System.getProperty('tests.coverage.html_report')) {
-      html.enabled true
-    }
+    // Code coverage report in HTML and CSV formats are on demand, in case they take extra disk space.
+    if (!System.getProperty('tests.coverage.report.html'))
+      html.enabled false
+    if (System.getProperty('tests.coverage.report.csv'))
+      csv.enabled true
```

Comment:
The comment says that these are on-demand, but the code tells me that by default `html.enabled=true` and `csv.enabled=false`. Where are those defaults? 

I'd expect this to be something like this:

```
xml.enabled System.getProperty('tests.coverage.report.xml', true)
html.enabled System.getProperty('tests.coverage.report.html', false)
csv.enabled System.getProperty('tests.coverage.report.html', false)
```

Reply:
Ah, you got a great idea! Thank you! It makes the code much clearer. 😄

In addition, if the "reports" configurations are not explicitly written, report in `xml` and `html` format will be generated, so it means this is the default behavior for "JaCoCo Gradle plugin".
```
reports {
    xml.enabled ...
    html.enabled ...
    csv.enabled ...
}
```